seahorn/include/utils.h

```diff
@@ -14,8 +17,8 @@ extern void klee_assume(uintptr_t condition);
 
 #ifdef __KLEE__ // define macros for KLEE_ASSUME, the assumption for KLEE only
 #define KLEE_ASSUME(X) klee_assume(X)
-#else 
-#define KLEE_ASSUME(X) 
```

Comment:
@LinerSu I needed to expand this to normal `assume` during fuzzing execution, otherwise compiler complains, let me know if you are fine with this change.

Reply:
there should be a define for fuzz compilation. You can expand KLEeassume macro the way you want under it. 

We already have define for seahorn and klee compilation